packages/smart-contracts/README.md

```diff
@@ -206,6 +206,26 @@ yarn hardhat deploy-live-payments --network private --force
 yarn hardhat deploy-live-payments --network private --force --dry-run
 ```
 
+## Administrate the contracts
+
+The contracts to be updated are listed in the array `create2ContractDeploymentList` in [Utils](scripts-create2/utils.ts).
+Modify the content of the array depending on your need when you perform an administration task.
+Environment variables needed: `ADMIN_PRIVATE_KEY`
+
+To update the contracts on one network, use:
+
+```bash
+NETWORK=<NETWORK> yarn hardhat update-contracts
```

Comment:
The network param already exists for every command, but we can only specify one. When we do it picks up the config defined for the network in the hardhat file.

That's why we use the env variable for deploying/administrating on several networks at once. This is from the xDeployer config. Maybe we could improve that, but it would be another PR.